npm_and_yarn/lib/dependabot/npm_and_yarn/package_manager.rb

```diff
@@ -256,21 +253,12 @@ def setup(name)
         end
 
         version ||= requested_version(name)
+        version ||= guessed_version(name).to_s
 
         if version
           raise_if_unsupported!(name, version)
-
           install(name, version)
-        else
-          version = guessed_version(name)
-
-          if version
-            raise_if_unsupported!(name, version.to_s)
-
-            install(name, version) if name == PNPMPackageManager::NAME
-          end
         end
```

Comment:
We want to install npm, and yarn versions as well. We removed the condition that only install for pnpm when version is guessed.